inc/class-spotim-options.php

```diff
@@ -419,6 +419,7 @@ function get_next_cron_execution( $timestamp ) {
         $registered_schedules = wp_get_schedules();
         if ( ! empty( $registered_schedules ) ) {
             foreach ( $registered_schedules as $key => $value ) {
+                unset( $value );
```

Comment:
I think if you want to unset value, then you should need to do:
```unset( $registered_schedules[ $key ] );```

Just want to know if I'm missing something.

Reply:
`unset( $registered_schedules[ $key ] );` will make the whole array empty, not that it's an issue but not needed, as `$value` is not being used in the loop. So that particular variable is only unset.

Reply:
if `$value` is not being used, then I think we don't need to anything, not unset or anything...

Reply:
If we don't unset, it'll throw a warning.